### PR TITLE
Allow OneLogin scaling fix JS to be enabled via local config flag

### PR DESCRIPTION
**Pull request by csduarte**
#### Summary
This PR adds a new a property to the config.json called `EnableOneLoginScalingFix`. Setting this property to true in the override.json will allow for specific JavaScript code to be injected into the SSO screen to adjust the OneLogin SSO form.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Added or updated unit tests (required for all new features)
- [ ] All new/modified APIs include changes to [mattermost-redux](https://github.com/mattermost/mattermost-redux) (please link)
- [ ] Has UI changes
- [ ] Includes text changes and localization file updates

@jarredwitt will coordinate modifications

### CODE REVIEW

**Comment by jarredwitt** [comment]
@enahum removed from config and only injected if the url includes `onelogin.com`

File: app/screens/sso/sso.js [hunk 1]
```diff
@@ -20,7 +20,20 @@ import StatusBar from 'app/components/status_bar';
 import PushNotifications from 'app/push_notifications';
 import {changeOpacity, makeStyleSheetFromTheme} from 'app/utils/theme';
 
-const jsCode = 'window.postMessage(document.body.innerText)';
+const postMessageJS = 'window.postMessage(document.body.innerText);';
```
Comment (enahum) [inline app/screens/sso/sso.js]: This code changed for release 1.3 perhaps we should wait until that is merged back into master?

File: app/screens/sso/sso.js [hunk 2]
```diff
@@ -168,12 +192,12 @@ class SSO extends PureComponent {
                     source={{uri: this.loginUrl}}
                     javaScriptEnabledAndroid={true}
                     automaticallyAdjustContentInsets={false}
-                    scalesPageToFit={true}
+                    scalesPageToFit={false}
```
Comment (enahum) [inline app/screens/sso/sso.js]: Does this causes changes in the way other sso pages display?

Comment (jarredwitt) [reply app/screens/sso/sso.js]: Moved this setting to state to make sure it only changes if SSO is OneLogin

**Pull request merged**